src/main/java/org/fogbowcloud/saps/engine/core/archiver/Archiver.java

```diff
@@ -45,8 +45,8 @@ protected Archiver(Properties properties, Catalog catalog) throws SapsException
         this.catalog = catalog;
         this.sapsExecutor = Executors.newScheduledThreadPool(1);
         this.permanentStorage = createStorageInstance(properties.getProperty(SapsPropertiesConstants.SAPS_PERMANENT_STORAGE_TYPE));
```

Comment:
I think we can clear the code by parsing and extracting all the variables defined in the properties file. So, for example, we avoid using the properties object in the `start` method and alike